yarn.lock

```diff
@@ -46,6 +46,14 @@
     "@angular-devkit/core" "11.0.1"
     rxjs "6.6.3"
 
+"@angular-devkit/architect@^0.1001.0":
```

Comment:
Seems like a lot of changes here, can we update only the necessary things?

Reply:
Fixed.